code/game/objects/items/defibrillator.dm

```diff
@@ -72,7 +72,7 @@
 		return FALSE
 	for(var/g in GLOB.observer_list)
 		var/mob/dead/observer/ghost = g
-		if(ghost.mind.current != src)
+		if(ghost?.mind.current != src)
```

Comment:
```suggestion
		if(ghost.mind.current != src)
```

Reply:
Well considering this is the entire PR, this suggestion would close it automatically

Reply:
Oh wait...
Hold on, why is there a `null` element in `observer_list`?

Reply:
It's an observer with a null mind, not a null element in observer_list